<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComWidget</class>
 <widget class="QWidget" name="ComWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,100">
   <item row="0" column="0">
    <widget class="QLabel" name="serialPortInfoLabel">
     <property name="text">
      <string>C&amp;OM:</string>
     </property>
     <property name="buddy">
      <cstring>serialPortInfoListBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="serialPortInfoListBox"/>
   </item>
   <item row="0" column="2" rowspan="9">
    <widget class="MonitorWidget" name="wMonitor" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="baudRateLabel">
     <property name="text">
      <string>&amp;BaudRate:</string>
     </property>
     <property name="buddy">
      <cstring>baudRateBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="baudRateBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dataBitsLabel">
     <property name="text">
      <string>Data bits:</string>
     </property>
     <property name="buddy">
      <cstring>dataBitsBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="dataBitsBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="parityLabel">
     <property name="text">
      <string>Parity:</string>
     </property>
     <property name="buddy">
      <cstring>parityBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="parityBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="stopBitsLabel">
     <property name="text">
      <string>Stop bits:</string>
     </property>
     <property name="buddy">
      <cstring>stopBitsBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="stopBitsBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="flowControlLabel">
     <property name="text">
      <string>Flow control:</string>
     </property>
     <property name="buddy">
      <cstring>flowControlBox</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="flowControlBox"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="btnConnect">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Connect</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btnDisconnect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Disconnect</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="manufacturerLabel">
       <property name="text">
        <string>Manufacturer:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="serialNumberLabel">
       <property name="text">
        <string>Serial number:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string>Location:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="vidLabel">
       <property name="text">
        <string>Vendor ID:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pidLabel">
       <property name="text">
        <string>Product ID:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actDisconnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MonitorWidget</class>
   <extends>QWidget</extends>
   <header location="global">monitorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialPortInfoListBox</tabstop>
  <tabstop>baudRateBox</tabstop>
  <tabstop>dataBitsBox</tabstop>
  <tabstop>parityBox</tabstop>
  <tabstop>stopBitsBox</tabstop>
  <tabstop>flowControlBox</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnDisconnect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
